### PR TITLE
Fix for issue #72: Refinded IRenderer interface

### PR DESCRIPTION
**Pull request by edgarmueller**

### CODE REVIEW

File: app/js/LocalController.js [hunk 1]
```diff
@@ -79,26 +79,22 @@ angular.module('makeithappen').controller('LocalController', ['$scope', function
...
     };
     $scope.usersUiSchema = {
-        "elements": [
+        "type": "Control",
+        "scope": {
+            "$ref": "#"
+        },
+        "columns": [
             {
-                "type": "Control",
```
Comment (sdirix) [inline app/js/LocalController.js]: The elements in the columns do not need "type":"Control" ? Maybe we should support any type of element there?

Comment (edgarmueller) [inline app/js/LocalController.js]: No, columns currently don't have any type (just like it was before this PR). What I've actually changed here it to remove the restriction that the top level element has no type attached, but an elements attribute must be present. Such a top-level entry point does not exist anymore, any UI element can be used now and if multiple elements at once should be specified a Layout should be used.

File: js/renderers/VerticalLayout.ts [hunk 2]
```diff
@@ -26,9 +25,19 @@ class VerticalLayout implements jsonforms.services.IRenderer {
         var renderedElements = renderElements(element.elements);
 
         return {
-            "type": "VerticalLayout",
+            "type": "Layout",
             "elements": renderedElements,
-            "size": 99
+            "size": 99,
+            "template":
+                `<fieldset class="col-sm-{{element.size}}">
```
Comment (sdirix) [inline js/renderers/VerticalLayout.ts]: Please remove the class from the fieldset. The size is already set on the parent div

File: js/renderers/HorizontalLayout.ts [hunk 3]
```diff
@@ -35,11 +34,23 @@ class HorizontalLayout implements jsonforms.services.IRenderer {
         }
 
         return {
-            "type": "HorizontalLayout",
+            "type": "Layout",
             "elements": renderedElements,
-            "size": maxSize
+            "size": maxSize,
+            "template":
+                `<fieldset class="col-sm-{{element.size}}">
```
Comment (sdirix) [inline js/renderers/HorizontalLayout.ts]: Please remove the class from fieldset

**Comment by sdirix** [comment]
Looks good ;)

**Comment by edgarmueller** [comment]
Thanks for the feedback, you're good to with merging :)

**Pull request merged**